test/utils/BaseRedeemablesTest.sol

```diff
@@ -208,4 +209,13 @@ contract BaseRedeemablesTest is RedeemablesErrors, BaseOrderTest {
             assertEq(IERC1155(address(token)).balanceOf(address(this), tokenId), 0);
         }
     }
+
+    function _mintToken(address token, uint256 tokenId, bool isToken721) internal {
+        if (isToken721) {
```

Comment:
can probably do a supportsInterface call to simplify not needing isToken721 bool